eng/CodeGeneration.targets

```diff
@@ -5,7 +5,7 @@
     <_AutoRestCoreVersion>3.0.6212</_AutoRestCoreVersion>
     <_AutoRestCSharpVersion>https://github.com/Azure/autorest.csharp/releases/download/1.0.0-dev.20200210.1/autorest-csharp-v3-1.0.0-dev.20200210.1.tgz</_AutoRestCSharpVersion>
     <_SupportsCodeGeneration Condition="'$(IsClientLibrary)' == 'true'">true</_SupportsCodeGeneration>
-    <AutoRestConfiguration>autorest.md</AutoRestConfiguration>
+    <AutoRestConfiguration Condition="'$(AutoRestConfiguration)' == ''">autorest.md</AutoRestConfiguration>
```

Comment:
This is a good change but why did you have to make it?

Reply:
I was trying to make the `swagger` directory a peer of `src` and `test` because I think there was an EngSys repo structure guideline somewhere suggesting that.  It was fighting me so I reverted `swagger` back under `src`, but left this here for the next brave soul who attempts it.